src/modules/cmdpal/Microsoft.CmdPal.UI.ViewModels/SettingsModel.cs

```diff
@@ -6,9 +6,12 @@
 using System.Text.Json;
 using System.Text.Json.Nodes;
 using System.Text.Json.Serialization;
+
 using CommunityToolkit.Mvvm.ComponentModel;
+
 using Microsoft.CmdPal.UI.ViewModels.Settings;
 using Microsoft.CommandPalette.Extensions.Toolkit;
+
```

Comment:
We should add `dotnet_separate_import_directive_groups = false` to the `.editorconfig` file.